api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoTrimestralCohortQueries.java

```diff
@@ -44,15 +44,14 @@ public CohortDefinition getA() {
     CohortDefinition transferredIn =
         getNumberOfPatientsTransferredInFromOtherHealthFacilitiesDuringCurrentMonth();
     CompositionCohortDefinition cd = new CompositionCohortDefinition();
-    cd.addParameter(new Parameter("onOrAfter", "onOrAfter", Date.class));
-    cd.addParameter(new Parameter("onOrBefore", "onOrBefore", Date.class));
-    cd.addParameter(new Parameter("location", "location", Location.class));
-    cd.addSearch("startedArt", mapStraightThrough(startedArt));
-    cd.addSearch(
-        "transferredIn",
-        map(
-            transferredIn,
-            "onOrAfter=${onOrAfter},onOrBefore=${onOrBefore+12m},location=${location}"));
+    cd.addParameters(getParameters());
+
+    String yearBefore =
+        "onOrAfter=${onOrAfter-12m},onOrBefore=${onOrBefore-12m},location=${location}";
```

Comment:
As the variable say, yearBefore, to me that being worked from the end of the reporting period, would be 
String yearBefore =
        "onOrAfter=${onOrBefore-12m},onOrBefore=${onOrBefore},location=${location}";

Reply:
sorry, that is coming from different class that might NOT be related to this PR

Reply:
Dear @ynurmahomed  please confirm if this now looks as it should.

Reply:
Looks like even this code is NOT supposed to be part of your commits, did you add them? seems it has caused discrepancies in another PR

Reply:
This is the way it is supposed to be as per #45. Somehow it got changed after that.

Reply:
All alterations to this were added from merging develop, however now @ynurmahomed requested this change was added.

Reply:
This changes would work better as part of a separate PR as it is not related to TX_TB and for historical purposes.